tests/test_file_readers.py

```diff
@@ -22,7 +22,7 @@ def test_read_csv_handles_headerless_columns_in_proper_number_and_order(runner):
         assert result_list[1]["header3"] == "col3_val2"
```

Comment:
Shouldn't there be some more tests for reading a csv, particularly when it only has some of the expected column headers?

Reply:
true

Reply:
these are now added